### ISSUE TITLE
Create web form for "Projects" 

### ISSUE THREAD

**Issue by bnvk** [opensourcedesign/opensourcedesign.github.io#102]
In the spirit on making it easier and clearer to contribute and engage with OSD and like our [add a job  form](http://opensourcedesign.net/jobs/job-form/) we should have a form for FOSS projects who are open to having designers join their community.

Ideally, there should be some criteria this form. Please chime in with ideas :smile:

**Comment by eupiteco** [comment]
Very interesting and useful! The first thing that comes to mind is to use a flag/radio button like the gratis/paid one. The options could be like job/project indicating whether the post is a single task (like logo or icon design) or if it is a project that is looking for designers to contribute and get involved. 
With this information I believer we can add a filter like the ones being discussed in #70.
What do you think?

**Comment by jancborchardt** [comment]
I’m not sure if it’s needed. There are already 2 places:

- If you are a project looking for a designer for a specific task, you submit a job to the job board.
- If you are a project open to designers (and have the relevant "getting started" guide for easy onboarding as well as use a "design" tag in your issue tracker), you can add it to the list at https://opensourcedesign.net/projects/ – just via a pull request like at https://github.com/opensourcedesign/opensourcedesign.github.io/pull/234

Because it seems that just a generic "hey we can use designers" might not be the best way to handle this. It’s always good to have clear expectations, either in the form of a specific job or a specific guide for designers.

**Comment by Erioldoesdesign** [comment]
I agree that the way for projects to recruit design contributors is via specific jobs (or even generalised jobs via the job form!) but this issue could be re-worded so that there is a form where people can submit projects to our https://opensourcedesign.net/projects/ page for the community to look at, review and include in the list.

I mentioned a comment from FOSDEM 2019 re. projects here: https://github.com/opensourcedesign/opensourcedesign.github.io/issues/125

@bnvk would you like to reword this issue to be about submitting projects to be included in the project page list or shall we close this one?

**Comment by jdittrich** [comment]
Closing this.
